README.md

```diff
@@ -1,24 +1,184 @@
 
-Things you may want to cover:
+  <img src="https://github.com/microverseinc/readme-template/raw/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
 
-* Ruby version
+</div>
 
-* System dependencies
+# 📗 Table of Contents
 
-* Configuration
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
 
-* Database creation
+<!-- PROJECT DESCRIPTION -->
 
-* Database initialization
+# 📖 BUDGET APP WITH ROR <a name="about-project"></a>
 
-* How to run the test suite
+> This is building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what. 
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+> The following tech stack was used:
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Ruby</a></li>
+    <li><a href="https://reactjs.org/">Ruby On Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Puma</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Style</summary>
+  <ul>
+    <li><a href="https://getbootstrap.com/docs/5.2/getting-started/introduction/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> The Following is how you can use the Blog App
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+> Ruby version : `ruby 3.1.2p20`
+    run `ruby -v`
+
+> SQLite3 version : `3.40.0`
+    run `sqlite3 --version`
+
+> npm version : `8.15.0`
+    run `npm -v`
+
+> yarn version : `1.22.19`
+    run `yarn -v`
+
+> Database : **PostgreSQL**
+### Setup
+
+Clone this repository to your desired folder:
+
+> `git clone git@github.com:David-Kasilia/budget_app_ror.git`
+
+### Install
+
+Install this project with:
+
+> `bundle install`
+
+### Usage
+
+To run the project, execute the following command:
+
+`rails server` or `rails s` 
+### Run tests
+
+To run tests, run the following command:
+`rspec spec`
+
+
+### Deployment
+
+You can deploy this project using:
+> Render <br/>
+> Here is a link to the deployment: https://budget-app-5474.onrender.com
```

Comment:
- [ **OPTIONAL** ] I think to make your deployment looks smart and professional, I suggest you write like in the example below, so it will only show text you hyperlink on it and not the full link.
```
Here is a (link to the deployment([https://budget-app-5474.onrender.com]
- [ **OPTIONAL** ]  I also suggest you copy this link in the `## Live Demo` section as it his expected section where it should be.